src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -1,5 +1,13 @@
 {% use "bootstrap_3_layout.html.twig" %}
 
+{% block form_label_class -%}
+col-sm-2
+{%- endblock form_label_class %}
```

Comment:
Can you move back this snippet to where it was before? That would ease merging older branches. Thanks.